index.bs

```diff
@@ -6026,8 +6026,9 @@ This section defines an additional set of extensions to be registered in the
 IANA "WebAuthn Extension Identifiers" registry [[!IANA-WebAuthn-Registries]] established by [[!RFC8809]].
```

Comment:
Please amend the line above this (6025) to:

```
This section and its subsections define an additional set of extensions to be registered in the
```

Reply:
(The "defines" still needs to be updated to "define" here :slightly_smiling_face:)